OxfordBodleian/Juel-Jensen/BDLaethg40.xml

```diff
@@ -231,7 +231,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 </item>
                                 
                                 <item xml:id="e3">
-                                    <desc>The last visible line contains <!--cursive--> Brillenbuchstaben.</desc>
+                                    <desc>The last visible line contains <hi rendition="simple:italic">Brillenbuchstaben</hi>.</desc>
```

Comment:
There was a Bm issue a while ago about how to refer to these – I think we settled on "magic symbols (charactêres)" but cannot find the issue now. Maybe we don't need to consider this.

Reply:
It's here https://github.com/BetaMasaheft/Manuscripts/pull/2777